### ISSUE TITLE
How to use the output file .gb to assign functions to ORFs.

### ISSUE THREAD

**Issue by manuss11** [deprekate/PHANOTATE#5]
Hello,
Thanks for developing this tool. I completed the annotation of some phages, but I don't know how to assign functionality to those ORFs. When I try to use the genbank file in RAST or PROKKA, those programs recall ORFs, so PHANOTATE info is not preserved.

Thanks a lot

**Comment by carolzhou** [comment]
Try using multiPhATE. You can find this tool at https://github.com/carolzhou/multiPhATE.

Sent from my iPhone

On May 3, 2019, at 2:01 AM, manuss11 <notifications@github.com<mailto:notifications@github.com>> wrote:


Hello,
Thanks for developing this tool. I completed the annotation of some phages, but I don't know how to assign functionality to those ORFs. When I try to use the genbank file in RAST or PROKKA, those programs recall ORFs, so PHANOTATE info is not preserved.

Thanks a lot

**Comment by manuss11** [comment]
Thanks a lot, I will try.

Regards

________________________________
De: Carol Zhou <notifications@github.com>
Enviado: Friday, May 3, 2019 6:41:57 PM
Para: deprekate/PHANOTATE
Cc: manuss11; Author
Asunto: Re: [deprekate/PHANOTATE] How to use the output file .gb to assign functions to ORFs. (#5)

Try using multiPhATE. You can find this tool at https://github.com/carolzhou/multiPhATE.

Sent from my iPhone

On May 3, 2019, at 2:01 AM, manuss11 <notifications@github.com<mailto:notifications@github.com>> wrote:


Hello,
Thanks for developing this tool. I completed the annotation of some phages, but I don't know how to assign functionality to those ORFs. When I try to use the genbank file in RAST or PROKKA, those programs recall ORFs, so PHANOTATE info is not preserved.

Thanks a lot

**Comment by deprekate** [comment]
yep, PHANOTATE is to get where the genes are, and multiPhATE is to get what they are : )
